examples/pylab_examples/image_origin.py

```diff
@@ -1,25 +1,25 @@
-#!/usr/bin/env python
 """
 You can specify whether images should be plotted with the array origin
 x[0,0] in the upper left or upper right by using the origin parameter.
 You can also control the default be setting image.origin in your
 matplotlibrc file; see http://matplotlib.org/matplotlibrc
 """
-from pylab import *
+import matplotlib.pyplot as plt
+import numpy as np
 
-x = arange(100.0)
+x = np.arange(100.0)
 x.shape = (10, 10)
 
 interp = 'bilinear'
-#interp = 'nearest'
+# interp = 'nearest'
 lim = -2, 11, -2, 6
-subplot(211, axisbg='g')
-title('blue should be up')
-imshow(x, origin='upper', interpolation=interp)
-#axis(lim)
+plt.subplot(211, axisbg='g')
+plt.title('blue should be up')
+plt.imshow(x, origin='upper', interpolation=interp)
```

Comment:
Future-proofing: this example depends on the default colormap of jet. Since that is going to change soon, how about we explicitly set that colormap now? Just add a `cmap='jet'` argument to the two imshow()'s